go.mod

```diff
@@ -1,12 +1,13 @@
 module app
 
-go 1.16
+go 1.18
```

Comment:
since we are not using anything specific from this version, what do you think about leaving this change for another PR?

Reply:
agree, also we might need to wait until we're able to compile with 1.18, https://catalog.redhat.com/software/containers/ubi8/go-toolset/5ce8713aac3db925c03774d1?container-tabs=overview is still on 1.16

Reply:
- In RHEL/Fedora/UBI there are no 1.18 RPM packages yet, we'd need custom UBI-based build image and install go 1.18 binaries manually. Or wait on Fedora 36, which is going out later this month, should contain golang 1.18 packages and base the build image on Fedora
- Or try to replace UBI-golang build image with upstream https://hub.docker.com/_/golang

Reply:
The second option could be quite good, unless there will be any problem/incompatibility

Reply:
And if we switch to 1.18 let's do the change in separate PR, which can be merged before this one

Reply:
seems, like simple replace of builder image works

Reply:
@jdobes #23